JavaScript_Advance/BOM_functions.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-multi-spaces */
+/* eslint-disable no-unused-vars */
+
+// window is the global variable for Browser environment
+// to achive it user just need to type window
+
+// sceen object contains main information about user device screen
+// in modern world sites mostly relate to adaptive layout and rarery use such client information
+const {
+  availHeight,
+  availLeft,
+  availTop,
+  availWidth,
+  colorDepth,
+  height,
+  orientation,
+  pixelDepth,
+  width
+} = window.screen;
+
+// window location is object provides interface and API to handle client url
+const {
+  assign,    /** method to manual switch to another url location.assign(path_to_page) */
```

Comment:
@eLeontev the assign is blue but why not all other things below it

Reply:
looks like here is issue with comments view, let me update it tomorrow with jsDoc format, also I will ad the similar info in doc file.